### PR TITLE
Decoupling from Azure Account extension

### PR DESCRIPTION
**Pull request by scale-tone**
#178

### CODE REVIEW

File: durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts [hunk 1]
```diff
@@ -171,48 +171,29 @@ export class TaskHubsCollector {
     }
 
     // Creates a user-specific access token for accessing Storage, also adds other needed headers
-    private async CreateIdentityBasedAuthHeadersForTableStorage(tokenCredential: any): Promise<{}> {
+    private async CreateIdentityBasedAuthHeadersForTableStorage(azureSubscription: AzureSubscription): Promise<{}> {
 
-        let token = '';
+        // Looks like azureSubscription.credential.getToken(scopes) does not respect scopes (getting a token with default 'https://management.core.windows.net' audience instead of Storage-specific scope) 
+        // So will have to use vscode.authentication directly
```
Comment (davidmrdavid) [inline durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts:177]: interesting - do you know if this is a known bug somewhere?

**Review by davidmrdavid — COMMENTED** [review]
In broad strokes the changes make sense to me, thank you for taking the lead here. I left some questions, and will look to validate myself that it works on a manual build and run

File: durablefunctionsmonitor-vscodeext/package.json [hunk 2]
```diff
@@ -30,13 +30,13 @@
...
     "activationEvents": [
         "onView:durableFunctionsMonitorTreeView",
-        "onCommand:extension.durableFunctionsMonitor",
-        "onCommand:extension.durableFunctionsMonitorPurgeHistory",
-        "onCommand:extension.durableFunctionsMonitorCleanEntityStorage",
-        "onCommand:extension.durableFunctionsMonitorBatchOps",
+        "onCommand:durable-functions-monitor.durableFunctionsMonitor",
+        "onCommand:durable-functions-monitor.durableFunctionsMonitorPurgeHistory",
+        "onCommand:durable-functions-monitor.durableFunctionsMonitorCleanEntityStorage",
+        "onCommand:durable-functions-monitor.durableFunctionsMonitorBatchOps",
```
[the remark this answers is not in the document]
Comment (scale-tone) [reply durablefunctionsmonitor-vscodeext/package.json:36]: This was my original mistake (5 years back) - to blindly copy the command prefix from some demo code. Didn't realize it has to be extension-specific.
Now we do have to rename it - to make sure it does not interfere with any other extension.
The worst thing can happen is people would have to reconfigure their keyboard shortcuts (if anyone has any).

File: durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts (hunk 1 above)
Comment (scale-tone) [reply durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts:177]: Turns out it is: https://github.com/microsoft/vscode-azuretools/issues/1596

**Review by davidmrdavid — APPROVED** [review]

File: durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts (hunk 1 above)
Comment (davidmrdavid) [reply durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts:177]: Makes sense. My main suggestion would be to link the GitHub issue there in the comments, but it's non-blocking

Comment (scale-tone) [reply durablefunctionsmonitor-vscodeext/src/TaskHubsCollector.ts]: Done. Thanks!

**Pull request merged**